R/read_capacity_factors.R

```diff
@@ -62,12 +63,13 @@ read_capacity_factors <- function(path = NULL,
     )
 
   } else {
+    # ADO 2393 - this should keep the source and filter based on global settings
+    # not in this hard coded manner
     data <- data %>%
       dplyr::select(
         .data$scenario, .data$scenario_geography, .data$technology,
         .data$year, .data$capacity_factor
       ) %>%
```

Comment:
maybe pipe in report_all_duplicate_kinds if in any thinkable way needed

Reply:
will add here. in these functions around the extending and wrangling the scenarios, I also think it would be good to look at passing the scenario source soon. We get that for pacta now and we would also be able to get that from all scenario related files. This would make it more robust.